#横向控制相关参数
k_pure: 0.3 # PurePursuit"增益系数
k_cte: 100  #Stanley"增益系数


#LQR_dynamics Q R矩阵参数
Q_ed: 20.0
Q_ed_dot: 1.0
Q_ephi: 10.0
Q_ephi_dot: 1.0
R_value: 40.0

#LQR_kinematics Q R矩阵参数
Q_ex_k: 3.0
Q_ed_k: 3.0
Q_ephi_k: 1.5
R_value_k: 4.0


#纵向控制PID
kp: 0.5
ki: 0.02
kd: 0.05

